viz/visualize.py

```diff
@@ -11,13 +11,18 @@
 from viz.viz_helpers import (reorder_img, read_loss_from_file, add_labels,
                              upsample, make_grid_img)
 
+import PIL
+from PIL import Image
```

Comment:
Either `import PIL` or import the exact things you need from PIL, but don't import `Image` twice